package.json

```diff
@@ -53,102 +53,89 @@
     "serve:angular": "nx serve angular",
     "prepack": "yarn build:core && yarn build:all",
     "test": "nx run-many --target=test --all",
-    "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main && yarn build:core",
-    "ng": "nx"
+    "postinstall": "ngcc --properties es2020 browser module main"
```

Comment:
Really happy we don't have to run the the cli script anymore ha! Has the script been deleted too?

Reply:
Similar to https://github.com/near/wallet-selector/pull/354#discussion_r924590855